libs/designsystem/src/lib/components/form-field/affix/input-affix.component.scss

```
@@ -0,0 +1,18 @@
+@import '~@kirbydesign/core/src/scss/base/variables';
```

Import other Sass files with `@use` instead of `@import` (Sass modules)

```suggestion
@use '../form-field-inputs.shared' as shared;
```

(change the relative path if I got the wrong number of `..`s 🙂)